glib/src/variant.rs

```diff
@@ -244,15 +244,27 @@ impl Variant {
     ///
     /// * if `self` is not a container type.
     /// * if given `index` is larger than number of children.
-    #[doc(alias = "get_child_value")]
-    #[doc(alias = "g_variant_get_child_value")]
```

Comment:
Why?

Reply:
It was moved below.
But the CI complains about this too, so I've restored it back and left the new helper without aliases.